sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/BeamSqlUdfExpression.java

```diff
@@ -78,6 +79,7 @@ private void reConstructMethod() {
       for (String pc : paraClassName) {
         paraClass.add(Class.forName(pc));
       }
+      udfIns = Class.forName(className).newInstance();
```

Comment:
It returns a new class instance every time. Let me include a logic to set it as null for static eval in another PR, although it doesn't take too much resource,